### PR TITLE
Adding managed ingestion and auth to charts. 

### PR DESCRIPTION
**Pull request by jjoyce0510**
This PR introduces support for default managed ingestion, along with authentication into the DataHub OSS Helm Charts. 

Specifically, there are a few new secrets that are introduced, along with a host of new environment variables, set inside of each deployment.yaml. 

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ datahub-datahub-frontend-84c58df9f7-5bgwx          1/1     Running     0
 datahub-datahub-gms-58b676f77c-c6pfx               1/1     Running     0          4m2s
 datahub-datahub-mae-consumer-7b98bf65d-tjbwx       1/1     Running     0          4m3s
 datahub-datahub-mce-consumer-8c57d8587-vjv9m       1/1     Running     0          4m2s
+acryl-datahub-actions-58b676f77c-c6pfx             1/1     Running     0          4m2s
```
Comment (dexter-mh-lee) [inline README.md:106]: is this name correct? why does this one not start with datahub?

Comment (dexter-mh-lee) [reply README.md:106]: where datahub is the name of the helm release

Comment (jjoyce0510) [reply README.md:106]: oh yes

File: charts/datahub/subcharts/datahub-mae-consumer/templates/deployment.yaml [hunk 2]
```diff
@@ -146,6 +146,17 @@ spec:
             {{- end }}
             - name: DATAHUB_ANALYTICS_ENABLED
               value: "{{ .Values.global.datahub_analytics_enabled }}"
+            {{- if .Values.global.datahub.metadata_service_authentication.enabled }}
```
Comment (dexter-mh-lee) [inline charts/datahub/subcharts/datahub-mae-consumer/templates/deployment.yaml:149]: I think maeconsumer also needs the managed-ingestion fields

**Review by dexter-mh-lee — APPROVED** [review]
LGTM!!!

File: charts/datahub/subcharts/datahub-mae-consumer/templates/deployment.yaml [hunk 3]
```diff
@@ -146,6 +146,24 @@ spec:
...
+                  name: {{ .Values.global.datahub.metadata_service_authentication.systemClientSecret.secretRef }}
+                  key: {{ .Values.global.datahub.metadata_service_authentication.systemClientSecret.secretKey }}
+            {{- end }}
+            {{- if .Values.global.datahub.managed_ingestion.enabled }}
+            - name: UI_INGESTION_ENABLED
+              value: "true"
+            {{- if .Values.global.datahub.managed_ingestion.defaultCliVersion }}
+            - name: UI_INGESTION_DEFAULT_CLI_VERSION
+              value: "{{ .Values.global.datahub.managed_ingestion.defaultCliVersion }}"
+            {{- end }}
```
Comment (dexter-mh-lee) [inline charts/datahub/subcharts/datahub-mae-consumer/templates/deployment.yaml:166]: Think we need an extra {{- end }}

**Pull request merged**